links/src/commonMain/kotlin/com/splendo/kaluga/links/models/LinksHandler.kt

```diff
@@ -15,14 +15,14 @@
 
  */
 
-package com.splendo.kaluga.links.manager
+package com.splendo.kaluga.links.models
 
 /**
- * Corresponds to urlParameterName and urlParameterValue.
+ * Corresponds to a map of urlParameterValue grouped by urlParameterName.
  */
-typealias NameValue = Pair<String, Any>
+typealias ParametersNameValue = Map<String, String>
 
 interface LinksHandler {
     fun isValid(url: String): Boolean
-    fun extractQueryAsList(url: String): List<NameValue>
+    fun extractQueryAsList(url: String): ParametersNameValue
```

Comment:
Should be renamed into `extractQuery` or `extractQueryAsMap`

Reply:
c2adaea